docs/tagged-literals.md

```diff
@@ -0,0 +1,137 @@
+# Literals with language tags
+
+?> New in 1.0
```

Comment:
@bergos I assume we're approaching 1.0?

Reply:
Yes, actually the idea was to release the 1.0 when we have the docs. Maybe we start with _new in_ after 1.0 and release 1.0 after the current batch of PRs?

Reply:
Your call. I assumed that a bunch of people may already have looked and used 0.12 and this would already be useful